README.md

```diff
@@ -65,6 +65,21 @@ After installing the JMeter Plugins Manager, the Correlation Recorder plugin wil
 - In the search field write Correlation Recorder, and then select the *bzm - Correlation Recorder* plugin’s checkbox
 - Click the button Apply Changes and Restart JMeter
 
+### Using it as a Maven dependency
+
+To use it as a Maven dependency include in `pom.xml`:
```

Comment:
I think we should paraphrase this to something like `If you use maven, just include following dependency:`